src/main/java/org/damap/base/enums/EDataAccessType.java

```diff
@@ -46,4 +46,20 @@ public static EDataAccessType getByValue(String value) {
       MAP.put(type.getValue(), type);
     }
   }
+
+  public int compare(EDataAccessType other) {
```

Comment:
We can generate the doc string for missing methods via a maven command.
Either way, a description of what this compare exactly does what would be great.

Reply:
Yes true, i forgot, thanks for reminding me.